aiida_quantumespresso/cli/calculations/dos.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+"""Command line scripts to launch a `PwCalculation` for testing and demonstration purposes."""
+from __future__ import absolute_import
+import click
+
+from aiida.orm import RemoteData
+from aiida.cmdline.params import options, types
+from aiida.cmdline.utils import decorators
+
+from aiida_quantumespresso.cli.utils import options as options_qe
+
+# required inputs: code, input_parameters, (parent_calculation or parent_folder)
```

Comment:
Why are these comments here?

Reply:
Self notes and documentation for the readers. They are not essential, I can remove them.